lib/path.js

```diff
@@ -168,7 +168,7 @@ const win32 = {
       let path;
       if (i >= 0) {
         path = args[i];
-        validateString(path, 'path');
+        validateString(path, `args[${i}]`);
```

Comment:
This appears to affect the `win32` implementation only, but it should affect both `win32` and `posix`.